src/slicegpt/adapters/opt_adapter.py

```diff
@@ -177,6 +177,10 @@ def seqlen(self) -> int:
     def hidden_size(self) -> int:
         return self.config.hidden_size
 
+    @property
+    def intermediate_size(self) -> int:
+        return self.config.ffn_dim
```

Comment:
I think this is the right property to use for OPT but still can't make the tests for vanilla OPT slicing work. @nailimixaM @LianaMikael

Reply:
I initially added intermediate_size just so we can save it with the slicing config but now that we have a proper config with copied parameters from the original config, I think this isn't needed anymore. I can just remove it